### DISCUSSION TITLE
Is anyone reading the Discussions?

### DISCUSSION THREAD

**Discussion by squidjam** [vmware-clarity/core#270] [Q&A] [answered]
I just want to know if anyone is reading.... There's notes about the data grid being in demo or discussion, but can't find it anywhere.
Nor have I received response from two issues found for the, then "in discussion", cds-dropdown, or the fact that some parts of the documentation show function names and not code.
Can someone please tell me if either I'm posting in the wrong space or if I'm just shouting into the void?
Thanks

**Comment by squidjam** [comment]
boink

**Reply by bbogdanov** [reply, marked as answer]
Hi, @squidjam

Clarity Core in the last year did not receive much focus and it is kinda on hold. 

The datagrid development is close to a point of being ready for a technical preview but since the team focus is elsewhere everybody using the datagrid is pretty much on their own

I hope that gives you clarity.
